src/devtools/domains/history.js

```diff
@@ -33,12 +36,24 @@ class History {
       }
     }
 
-    return { ...newHistory, list }
+    return { ...newHistory, list, focused }
+  }
+
+  focus(_old, id) {
+    let focused = null
```

Comment:
i believe when defining variables that will definitely be updated later, simply going with `let focused` is preferred